browser_patches/send_telegram_message.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+const https = require('https');
+
+const TELEGRAM_CHAT_ID = "-1001225613794";
+
+(async () => {
+  const { TELEGRAM_BOT_KEY } = process.env;
+  if (!TELEGRAM_BOT_KEY) {
+    console.log('environment variable \'TELEGRAM_BOT_KEY\' is not set');
+    return;
+  }
+
+  const text = process.argv[2];
+  if (!text) {
+    console.log('Text not set!');
+    console.log('Usage: node send_telegram_message.js <text>');
+    return;
+  }
+
+  await sendTelegramMessage(TELEGRAM_BOT_KEY, text);
+  console.log('Telegram message sent successfully!');
+})().catch(error => {
+  console.log(`Failed to send Telegram message. Error: ${error}`);
+  process.exit(1);
```

Comment:
This looks like a change in behavior that's undesirable. Now, if Telegram is down or flaky in a request, our build bot script will exit early in a number of places. 

Let's either exit with 0, so we still get a build; or wherever you call this script in the build script do `… || true`.

It's OK if we don't get a telegram message, but we want our builds!

While Telegram is unlikley to be down, I said the same thing about Slack in build scripts in other places, and the day Slack was down—actually several times in a year—I had to modify a build script just to get the important build artifact out.